ksql-engine/src/main/java/io/confluent/ksql/function/udf/string/MaskKeepLeftKudf.java

```diff
@@ -24,6 +23,7 @@
         + " Default masking rules will replace all upper-case characters with 'X', all lower-case"
         + " characters with 'x', all digits with 'n', and any other character with '-'.")
 public class MaskKeepLeftKudf {
+  private final String udfName = Masker.getMaskUdfName(this);
```

Comment:
This variable cannot be uppercase nor static because 1. the UDF name uses the class annotation which cannot be obtained statically, and 2. checkstyle fails with more than 3 uppercase letters in non-static variables.

Reply:
Fun workaround possibility, you can declare a static final variable NAME inside `MaskKeepLeftKudf` and then reference it in the annotation as `@UdfDescription(name=MaskKeepLeftKudf.NAME, author = ...)`. If we followed this pattern we also wouldn't need to use reflection to get the annotation from the class.